client/src/components/GamePage.js

```diff
@@ -49,6 +52,7 @@ const GamePage = () => {
                 setLetters(shuffledLetters);
                 setBoxes(new Array(answerLetters.length).fill(null));
                 setStartTime(new Date().getTime()); // Set the start time
+                localStorage.setItem('lastPlayedTime', new Date().getTime()); // Update the last played time
```

Comment:
do we need getTime() if we only look at mm dd and year?

Reply:
Also, is this the right place to set last played time? We should be setting it once they win or lose the game

Reply:
Makes the parsing easier since we are using a full normal date time not a month day setup just makes it easier to deal with